.github/workflows/pr-automation.yml

```
@@ -133,3 +133,12 @@ jobs:
       spec: ${{ needs.parse-tags.outputs.spec}}
       matrix: ${{ needs.parse-tags.outputs.matrix}}
       is-pg-build: ${{ github.event.pull_request.base.ref == 'pg' }}
+
+  perform-it:
+    needs: [ parse-tags ]
+    if: success() && needs.parse-tags.outputs.its == 'true'
```

actionlint (1.7.4)

property "its" is not defined in object type {matrix: string; spec: string; tags: string}

(expression)